grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -26,30 +27,48 @@
 
     private List<Repository> mRepositoryItems;
     private RecyclerView mRecyclerView;
-    private RecyclerView.Adapter mAdapter;
+    private LinearLayoutManager mLayoutManager;
+    private RepositoryAdpter mAdapter;
+
+    private int mPage = 0;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        mRepositoryItems = new ArrayList<>();
+
+        mRecyclerView = (RecyclerView) findViewById(R.id.recycler_main_activity);
+        mRecyclerView.setHasFixedSize(true);
+        mLayoutManager = new LinearLayoutManager(this);
+        mRecyclerView.setLayoutManager(mLayoutManager);
+
         fetchRepositories();
     }
 
     private void fetchRepositories() {
         GithubApi api = ApiClient.getClient().create(GithubApi.class);
-        Call<Repositories> call = api.getRepositories("language:java", "Repositories", "100");
+        Call<Repositories> call = api.getRepositories("language:java", "Repositories", ++mPage + "", "100");
 
         final ProgressBar progress = (ProgressBar) findViewById(R.id.progress_main_activity);
```

Comment:
Since also access a number of times ,`progress` should define as a member variable.